elpy.el

```diff
@@ -3702,7 +3702,11 @@ description."
       (when err-info
         (mapconcat #'flymake-ler-text
                    err-info
-                   ", ")))))
+                   ", "))))
+  (when (and (fboundp 'flymake-diagnostics) (fboundp 'flymake-diagnostic-text))
+    (mapconcat #'flymake-diagnostic-text (flymake-diagnostics (point)) "\n")
+    )
+  )
```

Comment:
This whole function probably should use a `cond` form. Also, it should return a newline-separated or comma-separated string, not one or the other depending on the flymake version.

Oh, and in Lisp, closing parens are never alone on a line, they get lonely and scared, and we do not like scared little parens!

Otherwise, looks good to me :-)